### DISCUSSION TITLE
Help for Call Confirm

### DISCUSSION THREAD

**Discussion by hongkong8521997** [community/community#23876] [New to GitHub]
<p>Dear Sir,</p>
<p>after I download version 1.9.2.apk from <a href="https://github.com/samigehi/sgsoft/tree/master/callconfirm" class="inline-onebox" rel="noopener nofollow ugc">sgsoft/callconfirm at master · samigehi/sgsoft · GitHub</a> , BUT I can not Add Contacts from “Ask for these contacts” even  I set the permission of Contacts and Telephone,<br>
please help me to fix it, thanks a lot🙏<br>
Please reply me as soon as possible🙏</p>
<p>Sunny</p>

**Comment by jainamoswal** [comment]
<p>Welcome to GitHub support, but unfortunately this issue is not related to “GitHub” nor it’s any “service”.</p>
<p>So I request you to reach their support via making an issue in that repo.</p>
<p>Thanks<br>
~ Jainam</p>
